### PR TITLE
SEO optimization/ featured activity packs

### PR DESCRIPTION
**Pull request by eadams17**
## WHAT
fix mobile views for featured activity packs index and individual pack view pages

## WHY
we want to fix these issues flagged by Google's mobile-friendly test tool

## HOW
refactor a few components and add media queries

### Screenshots
(If applicable. Also, please censor any sensitive data)
Screen Shot 2023-01-20 at 1 34 23 PM
Screen Shot 2023-01-20 at 1 37 09 PM

### Notion Card Links
(Please provide links to any relevant Notion card(s) relevant to this PR.)
https://www.notion.so/quill/Make-Activity-Pack-Filter-Page-Mobile-Friendly-95efaae76652447988235a422510f36d
https://www.notion.so/quill/Make-Activity-Pack-Page-Mobile-Friendly-f48fa6a366a24519a19316794032b381

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? |  yes
Have you deployed to Staging? | yes
Self-Review: Have you done an initial self-review of the code below on Github? | yes
Spec Review: Have you reviewed the spec and ensured this meets requirements and/or matches design mockups? | yes

### CODE REVIEW

File: services/QuillLMS/app/assets/stylesheets/pages/unit_templates/unit_template_minis/unit_template_minis.scss [hunk 1]
```diff
@@ -207,7 +206,37 @@
...
+        .data-table-row-section {
+          &.pack-name-section {
+            .quill-tooltip-trigger {
+              width: 40vw !important;
+            }
+          }
+        }
+      }
+    }
+  }
```
Comment (emilia-friedberg) [inline services/QuillLMS/app/assets/stylesheets/pages/unit_templates/unit_template_minis/unit_template_minis.scss]: Isn't this no longer visible anyway?

File: services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/unit_templates_manager/unit_template_profile/unit_template_profile.tsx [hunk 2]
```diff
@@ -107,12 +109,35 @@ export class UnitTemplateProfile extends React.Component<RouteComponentProps, Un
...
+    if(!activities) { return }
+
+    return(
+      <section className="mobile-activities-list-section">
+        <h3>Activities</h3>
+        <ul>
+          {activities.map(activity => {
+            const { id, name } = activity
+            return <li><button className="interactive-wrapper" onClick={this.handleActivityClick} value={id}>{name}</button></li>
+          })}
```
Comment (emilia-friedberg) [inline services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/unit_templates_manager/unit_template_profile/unit_template_profile.tsx]: Should they just be links?

**Review by emilia-friedberg — APPROVED** [review]

File: services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/unit_templates_manager/unit_template_profile/unit_template_profile.tsx (hunk 2 above)
Comment (eadams17) [reply services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/unit_templates_manager/unit_template_profile/unit_template_profile.tsx:132]: @emilia-friedberg I could, but then I suppose I'd have to use an arrow function to call `redirectToActivity` in `onClick` in order to pass the id argument? (and just add a linter exception)

Comment (eadams17) [reply services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/unit_templates_manager/unit_template_profile/unit_template_profile.tsx]: Sure! I can change them to links

**Pull request merged**